pkg/mqtrigger/kafka/main.go

```diff
@@ -26,6 +26,13 @@ import (
 	"github.com/fission/fission/pkg/mqtrigger/util"
 )
 
+// Following code snippet is from KEDA project and adapted for Fission.
```

Comment:
Let's remove this from here along with rest of Kafka code and use fission/keda-connector for that

Reply:
yes, add that in fission/keda-connector